### PR TITLE
python3Packages.dask: 2022.01.0 -> 2022.02.0, python3Packages.distributed: 2021.12.0 -> 2022.2.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Fix build (https://hydra.nixos.org/build/167244566)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Depends on:

- #160001

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 160019` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages marked as broken and skipped:</summary>
  <ul>
    <li>python310Packages.dask-xgboost</li>
    <li>python39Packages.dask-xgboost</li>
    <li>python39Packages.rl-coach</li>
  </ul>
</details>
<details>
  <summary>40 packages failed to build:</summary>
  <ul>
    <li>dvc</li>
    <li>dvc-with-remotes</li>
    <li>python310Packages.aplpy</li>
    <li>python310Packages.asdf</li>
    <li>python310Packages.batchgenerators</li>
    <li>python310Packages.caffe</li>
    <li>python310Packages.dask-image</li>
    <li>python310Packages.env-canada</li>
    <li>python310Packages.glymur</li>
    <li>python310Packages.handout</li>
    <li>python310Packages.image-match</li>
    <li>python310Packages.imagecorruptions</li>
    <li>python310Packages.imageio</li>
    <li>python310Packages.imgaug</li>
    <li>python310Packages.intensity-normalization</li>
    <li>python310Packages.moviepy</li>
    <li>python310Packages.napari</li>
    <li>python310Packages.napari-console</li>
    <li>python310Packages.napari-svg</li>
    <li>python310Packages.pims</li>
    <li>python310Packages.pyregion</li>
    <li>python310Packages.pytest-astropy</li>
    <li>python310Packages.pytest-astropy-header</li>
    <li>python310Packages.reproject</li>
    <li>python310Packages.runway-python</li>
    <li>python310Packages.scikitimage</li>
    <li>python310Packages.scmrepo</li>
    <li>python310Packages.slicedimage</li>
    <li>python310Packages.spectral-cube</li>
    <li>python310Packages.sunpy</li>
    <li>python310Packages.tifffile</li>
    <li>python310Packages.tokenizers</li>
    <li>python310Packages.transformers</li>
    <li>python39Packages.aplpy</li>
    <li>python39Packages.asdf</li>
    <li>python39Packages.bpycv</li>
    <li>python39Packages.scmrepo</li>
    <li>python39Packages.spectral-cube</li>
    <li>python39Packages.sunpy</li>
    <li>python39Packages.zcs</li>
  </ul>
</details>
<details>
  <summary>75 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>intensity-normalization (python39Packages.intensity-normalization)</li>
    <li>napari (python39Packages.napari)</li>
    <li>python310Packages.casa-formats-io</li>
    <li>python310Packages.dask</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.datasets</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.fsspec</li>
    <li>python310Packages.gcsfs</li>
    <li>python310Packages.ibis-framework</li>
    <li>python310Packages.pyfftw</li>
    <li>python310Packages.s3fs</li>
    <li>python310Packages.streamz</li>
    <li>python39Packages.batchgenerators</li>
    <li>python39Packages.boxx</li>
    <li>python39Packages.caffe</li>
    <li>python39Packages.casa-formats-io</li>
    <li>python39Packages.clifford</li>
    <li>python39Packages.dask</li>
    <li>python39Packages.dask-gateway</li>
    <li>python39Packages.dask-glm</li>
    <li>python39Packages.dask-image</li>
    <li>python39Packages.dask-jobqueue</li>
    <li>python39Packages.dask-ml</li>
    <li>python39Packages.dask-mpi</li>
    <li>python39Packages.datasets</li>
    <li>python39Packages.datashader</li>
    <li>python39Packages.distributed</li>
    <li>python39Packages.env-canada</li>
    <li>python39Packages.fastparquet</li>
    <li>python39Packages.fsspec</li>
    <li>python39Packages.gcsfs</li>
    <li>python39Packages.glymur</li>
    <li>python39Packages.handout</li>
    <li>python39Packages.ibis-framework</li>
    <li>python39Packages.image-match</li>
    <li>python39Packages.imagecorruptions</li>
    <li>python39Packages.imageio</li>
    <li>python39Packages.imgaug</li>
    <li>python39Packages.intake</li>
    <li>python39Packages.intake-parquet</li>
    <li>python39Packages.mask-rcnn</li>
    <li>python39Packages.moviepy</li>
    <li>python39Packages.napari-console</li>
    <li>python39Packages.napari-svg</li>
    <li>python39Packages.pims</li>
    <li>python39Packages.pyfftw</li>
    <li>python39Packages.pyregion</li>
    <li>python39Packages.pytest-astropy</li>
    <li>python39Packages.pytest-astropy-header</li>
    <li>python39Packages.qiskit</li>
    <li>python39Packages.qiskit-machine-learning</li>
    <li>python39Packages.reproject</li>
    <li>python39Packages.runway-python</li>
    <li>python39Packages.s3fs</li>
    <li>python39Packages.scikitimage</li>
    <li>python39Packages.slicedimage</li>
    <li>python39Packages.spacy-transformers</li>
    <li>python39Packages.sparse</li>
    <li>python39Packages.streamz</li>
    <li>python39Packages.stumpy</li>
    <li>python39Packages.stytra</li>
    <li>python39Packages.tensorflow-datasets</li>
    <li>python39Packages.tensorly</li>
    <li>python39Packages.test-tube</li>
    <li>python39Packages.tifffile</li>
    <li>python39Packages.tokenizers</li>
    <li>python39Packages.transformers</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tts</li>
  </ul>
</details>
